src/eval-frame/actions/fetch-cell-eval-actions.js

```diff
@@ -99,71 +102,57 @@ export async function handleFetch(fetchInfo) {
 }
 
 export function evaluateFetchText(fetchText, evalId) {
-  return dispatch => {
```

Comment:
there is actually not much changed in this file, but the diff is garbage. the main thing is that this is no longer a redux-thunk, because it no longer needs to be dispatched -- it's just a plain function. the diff looks like a lot has changed b/c the indents of all these lines changed